src/_P036_FrameOLED.ino

```diff
@@ -761,22 +770,31 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
           }
 
 #ifdef PLUGIN_036_DEBUG
-          String log = String(F("[P36] Line: ")) + String(LineNo);
-          log += String(F(" NewContent:")) + String(NewContent);
-          log += String(F(" Content:")) + String(P036_data->DisplayLinesV1[LineNo - 1].Content);
-          log += String(F(" Length:")) + String(String(P036_data->DisplayLinesV1[LineNo - 1].Content).length());
-          log += String(F(" Pix: ")) + String(P036_data->display->getStringWidth(String(P036_data->DisplayLinesV1[LineNo - 1].Content)));
-          log += String(F(" Reserved:")) + String(P036_data->DisplayLinesV1[LineNo - 1].reserved);
+          String log = F("[P36] Line: ");
```

Comment:
These strings for logging can become quite long, and also use quite a lot of operator+
so please try to estimate the final length and use `reserve` on the string before adding content to it.

Reply:
Change in the next commit.